src/simulation/Simulation.hh

```diff
@@ -822,8 +822,7 @@ namespace sim {
 
             while(true) {
                 if (iteration >= 1000) {
```

Comment:
Is there a reason for hard-coding the value of 1000 here and not using `maxIterations`?

Reply:
No, this is probably some debugging artifact